source/globals.d.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: Drop some definitions when their related bugs are resolved
```

Comment:
I merged all of these TODOs into one. They're just noise since we can only wait.